core/trino-main/src/main/java/io/trino/server/protocol/Query.java

```diff
@@ -633,23 +633,23 @@ private String formatType(DataType type)
                 if (dataTimeType.getType() == DateTimeDataType.Type.TIMESTAMP && !dataTimeType.isWithTimeZone()) {
                     return TIMESTAMP;
                 }
-                else if (dataTimeType.getType() == DateTimeDataType.Type.TIME && !dataTimeType.isWithTimeZone()) {
+                if (dataTimeType.getType() == DateTimeDataType.Type.TIME && !dataTimeType.isWithTimeZone()) {
```

Comment:
?

Reply:
The "else" is redundant since the above "if" always returns. IntelliJ will warn and perform this conversion automatically (and safely).